source/upgrade.txt

```diff
@@ -102,6 +102,8 @@ Version 6.x Breaking Changes
   perform any operations in the session.
 - The ``includeResultMetadata`` option for compound operation methods is
   ``false`` by default.
+- Raised optional Kerberos dependency minimum version to 2.0.1 and
+  removed support for version 1.x.
```

Comment:
Issue:

Looks like the zstd optional dependency was also raised per https://github.com/mongodb/node-mongodb-native/releases/tag/v6.0.0

Suggestion:
Unless this is already represented in another ticket, I think it would be helpful to mention.